packages/core-mobile/tests/firebase/jestSetup.js

```diff
@@ -0,0 +1,2 @@
+jest.mock('@react-native-firebase/app-check', () => ({}))
```

Comment:
can these be moved to `core-mobile/__mocks__`?

Reply:
[Move firebase mocks to __mocks__](https://github.com/ava-labs/core-mobile/pull/1659/commits/148abb6376e1831f670d758ea200188b83f6a774)

Reply:
Actually it can't, tests are failing 
[Revert "Move firebase mocks to __mocks__"](https://github.com/ava-labs/core-mobile/pull/1659/commits/34905f25343a0eac0031d6ad10761ff7b9257d19)